cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -789,7 +789,16 @@ storage:
           # don't audit events from the system:apiserver user
           - level: None
             users: ["system:apiserver"]
-          # don't audit any kube-controller-manager events
+          # audit pod events from kube-controller-manager
+          - level: Request
+            users: ["system:kube-controller-manager"]
+            omitStages:
+              - "RequestReceived"
+            verbs: ["create". "delete", "update", "patch", "deletecollection"]
```

Comment:
Do we want to add `update` and `patch`, are they relevant? @aermakov-zalando had an opinion on this?

Reply:
By looking at the amount of events we would send (Pod events without GET,LIST,WATCH) it would be ok. But I'm also fine dropping them if they not necessary. I have no opinion on that.

Reply:
I would say yes, but it's not really necessary in the first iteration. You can't change compliance-relevant things because pods are immutable.